api/src/main/java/org/eclipse/microprofile/openapi/annotations/callbacks/Callback.java

```diff
@@ -77,6 +77,16 @@
      **/
     String ref() default "";
 
+    /**
+     * Reference value to a Path Item object, to be referenced by the Callback object.
+     * <p>
+     * This property provides a reference to a Path Item object defined elsewhere. {@code #callbackUrlExpression()} is
+     * REQUIRED when this property is set. The referenced Path Item will be used for the URL expression specified.
+     *
+     * @since 4.0
+     */
+    String pathItemRef() default "";
```

Comment:
It's hard to get my head around a `Callback` being a reference to either another `Callback` or a `PathItem`. Is it correct to say that our `Callback` is effectively a map of `PathItem`s? So, it's basically just one level above a `PathItem`, which is why using the `pathItemRef` requires the `callbackUrlExpression` to complete the "entry".

Reply:
Yep, that's pretty much it.

We could probably do with some examples in the class Javadoc, though we could do with examples on every annotation, along with saying where to use it and what it does.